crml/sylo/src/payment.rs

```diff
@@ -0,0 +1,141 @@
+/* Copyright 2020 Centrality Investments Limited
+*
+* Licensed under the LGPL, Version 3.0 (the "License");
+* you may not use this file except in compliance with the License.
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+* You may obtain a copy of the License at the root of this project source code,
+* or at:
+*     https://centrality.ai/licenses/gplv3.txt
+*     https://centrality.ai/licenses/lgplv3.txt
+*/
+
+//! Manage the authorized accounts set for the Sylo fee payment
+
+use frame_support::{decl_module, decl_storage, ensure, weights::SimpleDispatchInfo, IterableStorageMap};
+use frame_system::{ensure_root, ensure_signed};
+use sp_runtime::DispatchResult;
+use sp_std::prelude::*;
+
+pub trait Trait: frame_system::Trait {}
+
+const NOT_SYLO_PAYER: &str = "You are not a Sylo payer!";
+
+decl_storage! {
+	trait Store for Module<T: Trait> as SyloModule {
```

Comment:
```suggestion
	trait Store for Module<T: Trait> as SyloFeePayment {
```
or something more descriptive

Reply:
Done